Apps/APN/ios/Podfile

```
@@ -29,5 +27,5 @@ target 'SampleApp' do
 end
 
 target 'NotificationServiceExtension' do
-  pod 'CustomerIO/MessagingPushAPN', '~> 2'
+  pod 'customerio-reactnative-richpush/apn', :path => '../node_modules/customerio-reactnative'
```

I got this idea from other SDKs doing the same thing where you specify `:path` that points to the locally installed `node_modules` path for the RN app.

I like the solution because we want the RN SDK npm module to encapsulate all of the files needed to install our SDK successfully. After a customer installs a version of the RN SDK, this line of code automatically gets the hard-coded version of the native SDKs via the RN SDK npm package.

---

I asked it down below in a different context, but shouldn't this also work?

```suggestion
  pod 'customerio-reactnative/apn', :path => '../node_modules/customerio-reactnative'
```

since we have subspec there?

---

See [this comment](https://github.com/customerio/customerio-reactnative/pull/149/files#r1228350452) to explain why this isn't possible.